packages/ui/src/components/Tabs/api.ts

```diff
@@ -0,0 +1,22 @@
+import { forceInject } from '@/util'
+import { InjectionKey } from 'vue'
+
+export interface TabsPanelApi {
+  active: string
+  selectTab: (tab: string) => void
+  background: string
+}
+
+export type BackgroundType = 'primary' | 'secondary' | 'none'
+
+export enum BackgroundTypeValues {
+  'primary',
+  'secondary',
+  'none',
+}

```

Comment:
- `BackgroundType` is a very general name. This type is exported directly from the library, and it may confuse - which background? For what? Usually for such types we always use component prefix, like `TabsPanelBackgroundType`
- To avoid repetition, you can use this trick:

   ```ts
   export const TABS_PANEL_BACKGROUND_TYPES = ['primary', 'secondary', 'none'] as const
   
   export type TabsPanelBackgroundType = 
     typeof TABS_PANEL_BACKGROUND_TYPES extends ReadonlyArray<infer T>
       ? T
       : never
   ```